doc/release-notes/0.9/0.9.md

```diff
@@ -0,0 +1,228 @@
+<td valign="top">File <code>ibmjvmti.h</code> not found in OpenJ9 JDK</td>
+<td valign="top">All platforms</td>
+<td valign="top">JVMTI extensions cannot be compiled.</td>
+<td valign="top">None</td>
+</tr>
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/1252">#1252</a></td>
+<td valign="top">GPU acceleration with <code>java.util.Array.sort()</code> is not supported</td>
+<td valign="top">Linux and Windows platforms</td>
+<td valign="top">GPU acceleration on <code>sort()</code> operations is not available.</td>
+<td valign="top">None<!-- , although <code>com.ibm.gpu.Maths</code> can be used directly. --></td>
+</tr>
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/897">#897</a></td>
+<td valign="top">Javadoc not building some OpenJ9 extensions.</td>
+<td valign="top">All platforms</td>
+<td valign="top">Some API documentation is missing in the <code>build/&lt;platform&gt;/docs</code> directory.</td>
+<td valign="top">None</td>
+</tr>
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/898">#898</a></td>
+<td valign="top">Javadoc building some unsupported OpenJ9 extensions.</td>
+<td valign="top">All platforms</td>
+<td valign="top">Some API documentation in <code>build/&lt;platform&gt;/docs</code> is not supported with OpenJ9.</td>
+<td valign="top">None</td>
+</tr>
+<tr><td valign="top"><a href="https://github.com/ibmruntimes/openj9-openjdk-jdk8/issues/17">#17</a></td>
+<td valign="top">Builds contain extra shared libraries.</td>
+<td valign="top">All platforms</td>
+<td valign="top">Shared libraries must not be used in production.</td>
+<td valign="top">None</td>
+</tr>
+</tbody>
+</table>
+
+
+## Functional verification tests
+The following issues relate to test failures:
+
+- [#238](https://github.com/eclipse/openj9/issues/238), [#244](https://github.com/eclipse/openj9/issues/244), [#362](https://github.com/eclipse/openj9/issues/362), [#366](https://github.com/eclipse/openj9/issues/366), [#480](https://github.com/eclipse/openj9/issues/480), [#858](https://github.com/eclipse/openj9/issues/858), [#968](https://github.com/eclipse/openj9/issues/968), [#1127](https://github.com/eclipse/openj9/issues/1127), [#1128](https://github.com/eclipse/openj9/issues/1128), [#1129](https://github.com/eclipse/openj9/issues/1129), [#1130](https://github.com/eclipse/openj9/issues/1130), [#1131](https://github.com/eclipse/openj9/issues/1131), [#1144](https://github.com/eclipse/openj9/issues/1144), [#1151](https://github.com/eclipse/openj9/issues/1151), [#1203](https://github.com/eclipse/openj9/issues/1203), [#1204](https://github.com/eclipse/openj9/issues/1204), [#1205](https://github.com/eclipse/openj9/issues/1205), [#1207](https://github.com/eclipse/openj9/issues/1207), [#1209](https://github.com/eclipse/openj9/issues/1209), [#1212](https://github.com/eclipse/openj9/issues/1212), and [#1244](https://github.com/eclipse/openj9/issues/1244).
```

Comment:
Please update this list. Some of these are resolved, and there are new issues as well.